### PR TITLE
EZP-23869: Added dependency on legacy-bridge

### PR DESCRIPTION
**Pull request by bdunogier**
> Related to ezsystems/ezpublish-community#223

### CODE REVIEW

**Comment by yannickroger** [comment]
+1

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
         "php": ">=5.3.3",
         "ezsystems/ezpublish-kernel": "@dev",
         "ezsystems/comments-bundle": "@dev",
+        "ezsystems/legacy-bridge": "dev-master",
```
Comment (lolautruche) [inline composer.json]: Why not `@dev` ? :stuck_out_tongue_winking_eye:

**Comment by lolautruche** [comment]
+1

**Comment by bdunogier** [comment]
Trying to do without this pull-request. Maybe it can be closed.

**Comment by bdunogier** [comment]
Not needed anymroe.

**Pull request closed without merging**